templates/rhdm70-kieserver-s2i.yaml

```diff
@@ -0,0 +1,471 @@
+    description: Application template for Red Hat Decision Manager Execution Server 7.0 application built using S2I.
+    iconClass: icon-jboss
+    tags: rhdm,jboss,xpaas
+    version: 1.4.0
+    openshift.io/display-name: Red Hat Decision Manager Execution Server 7.0 S2I (Ephemeral with https)
+  name: rhdm70-kieserver-s2i
+labels:
+  template: rhdm70-kieserver-s2i
+  xpaas: 1.4.0
+message: A new Decision Manager Execution Server application has been created in your
+  project. Please be sure to create the "rhdm-service-account" service account and
+  the secret named "${HTTPS_SECRET}" containing the ${HTTPS_KEYSTORE} file used for
+  serving secure content.
+parameters:
+- displayName: EAP Admin User
+  description: EAP administrator username
+  name: ADMIN_USERNAME
+  value: eapadmin
+  required: false
+- displayName: EAP Admin Password
+  description: EAP administrator password
+  name: ADMIN_PASSWORD
+  from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: false
+- displayName: KIE Server Controller User
+  description: KIE server controller username (Sets the org.kie.server.controller.user
+    system property)
+  name: KIE_SERVER_CONTROLLER_USER
+  value: controllerUser
+  required: false
+- displayName: KIE Server Controller Password
+  description: KIE server controller password (Sets the org.kie.server.controller.pwd
+    system property)
+  name: KIE_SERVER_CONTROLLER_PWD
+  from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: false
+- displayName: KIE server controller protocol
+  description: KIE server controller protocol (Used to set the org.kie.server.controller
+    system property)
+  name: KIE_SERVER_CONTROLLER_PROTOCOL
+  value: http
+  required: false
+- displayName: KIE server controller service
+  description: KIE server controller service (Used to set the org.kie.server.controller
+    system property if host and port aren't set)
+  name: KIE_SERVER_CONTROLLER_SERVICE
+  value: ''
+  required: false
+- displayName: KIE server controller host
+  description: KIE server controller host (Used to set the org.kie.server.controller
+    system property)
+  name: KIE_SERVER_CONTROLLER_HOST
+  example: my-app-controller-ocpuser.os.example.com
+  required: false
+- displayName: KIE server controller port
+  description: KIE server controller port (Used to set the org.kie.server.controller
+    system property)
+  name: KIE_SERVER_CONTROLLER_PORT
+  example: '8080'
+  required: false
+- displayName: KIE Admin User
+  description: KIE administrator username
+  name: KIE_ADMIN_USER
+  value: adminUser
+  required: false
+- displayName: KIE Admin Password
+  description: KIE administrator password
+  name: KIE_ADMIN_PWD
+  from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: false
+- displayName: KIE Server Container Deployment
+  description: 'KIE Server Container deployment configuration in format: containerId=groupId:artifactId:version|c2=g2:a2:v2'
+  name: KIE_SERVER_CONTAINER_DEPLOYMENT
```

Comment:
Can be this env. variable moved closer to S2I source variables? This variable set what should be deployed on the Kie server and for me it is a more readable when these variables are together.